.cirrus.yml

```diff
@@ -19,12 +17,10 @@ task:
 task:
   name: Release
   only_if: $CIRRUS_TAG != ''
-  macos_instance:
-    image: ghcr.io/cirruslabs/macos-ventura-xcode:latest
   env:
+    AC_PASSWORD: ENCRYPTED[4a761023e7e06fe2eb350c8b6e8e7ca961af193cb9ba47605f25f1d353abc3142606f412e405be48fd897a78787ea8c2]
     GITHUB_TOKEN: ENCRYPTED[!98ace8259c6024da912c14d5a3c5c6aac186890a8d4819fad78f3e0c41a4e0cd3a2537dd6e91493952fb056fa434be7c!]
     GORELEASER_KEY: ENCRYPTED[!9b80b6ef684ceaf40edd4c7af93014ee156c8aba7e6e5795f41c482729887b5c31f36b651491d790f1f668670888d9fd!]
-  install_script: brew install go goreleaser/tap/goreleaser-pro
```

Comment:
I also reverted to always build on the worker because it was a little weird that we test in one environment but build binaries in another.

Agreed it's not ideal for reproducibility but at least test and build environments are the same.

Reply:
Hmm, I thought re-using the `ghcr.io/cirruslabs/macos-ventura-xcode:latest` provides the same build environment. I am missing something?

Reply:
It's not the same. We try to sync the `macos-ventura-xcode:latest` and our physical M1 Mac Mini to always have the latest Ventura and Xcode but the environments are not identical.